examples/image_border.py

```diff
@@ -0,0 +1,42 @@
+"""
+Layer Border Overlay
+====================
+
+Display an image in napari and add a border overlay (i.e. `bounding_box`).
+
+Demonstrates visualization of many border overlay properties,
```

Comment:
```suggestion
The bounding box overlay is a visual representation of the extents of a layer.
This example demonstrates visualization of many bounding box overlay properties,
```
I like the descriptor from the other example.